Mu2eG4/src/Mu2eSpecialCutsProcess.cc

```diff
@@ -65,15 +65,17 @@ namespace mu2e{
       proposedStep = 0.;
     }
     if (verboseLevel>0) {
+      G4int prec = G4cout.precision(15);
       G4cout << __func__ << " : "
              << GetProcessName()
              << " : current step "
              << aTrack.GetCurrentStepNumber()
              << ", max step limit "
              << static_cast<G4int>(mu2elimits_.maxStepsPerTrack())
              << ", proposed step "
-             << proposedStep
+             << std::setw(24) << std::scientific << proposedStep
```

Comment:
I am pretty sure that std::scientific is sticky.  So save and restore as you did for precision.  There are two places.

Reply:
Thanks, you are right. I added << std::defaultfloat